library/src/main/java/com/novoda/downloadmanager/domain/DownloadStatus.java

```diff
@@ -11,7 +11,6 @@
     public static DownloadStatus from(DownloadStage stage) {
         switch (stage) {
             case QUEUED:
-            case SUBMITTED:
```

Comment:
and this still holds true - a Download will go from QUEUED -> RUNNING.

The submitted state is a pseudo-state that only needs to exist as a concept for whoever is submitting the downloads - doesn't need to be persisted.